workspace-manager-clienttests/src/main/java/scripts/testscripts/ServiceStatus.java

```diff
@@ -9,11 +9,28 @@
 import org.slf4j.LoggerFactory;
 import scripts.utils.ClientTestUtils;
 
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
 public class ServiceStatus extends TestScript {
     private static final Logger logger = LoggerFactory.getLogger(ServiceStatus.class);
+    private Duration delay = Duration.ZERO;
+
+    @Override
+    public void setParameters(List<String> parameters) {
+
+        if (parameters == null || parameters.size() == 0) {
+            return;
+        }
+        delay = Duration.ofSeconds(Long.parseLong(parameters.get(0)));
+    }
 
     @Override
     public void userJourney(TestUserSpecification testUser) throws Exception {
+        if (delay.getSeconds() > 0) TimeUnit.SECONDS.sleep(delay.getSeconds());
```

Comment:
Why did you need this additional sleep parameter?

Reply:
The sleep parameter simulates an artificial delay to give time for the disruption script to kill the pods before executing the test script. The service status thread normally exits right the way because it takes little time to execute, before the disruption can take place.

Reply:
Ah understood. Could you also perhaps just run a lot of status checks (e.g. numberOfUserJourneyThreads = 500 or something large) so that some of them fail? Both are fine for a demo though.

Also, okay to do in a future PR -- there is a sleep in the delete pods disruptive script. It would be good to make that configurable -- then you probably wouldn't need this.

Reply:
Same thoughts here, I have added these suggestions to [Future Enhancement](https://docs.google.com/document/d/1gYqlJTpC4C4S1K0pYikeNJx-HWNZAZJYybinmuqcSTI/edit#) doc.